src/module-1/cards-list-v1/cardslist.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="../card/style.css">
+    <link rel="stylesheet" href="liststyle.css">
+</head>
+<body>
+    <div class="section" id="main">
+        <div class="container">
+            <div class="card">
+                <div class="card__wrapper">
+                    <div class="card__image-container">
+                        <img src="apple-watch.svg" alt="" class="card__image">
+                    </div>
+                    <div class="card__rating">
+                        <div class="card-rating__stars">
+                            <div class="card-rating__stars-container">
+                                3.4
+                            </div>
+                            <img src="star.svg" alt="" class="stars__star">
+                        </div>
+                        <div class="card-rating__price">$399</div>
+                    </div>
+                    <div class="card__description">
+                        <div class="card-description__title">Apple Watch Series 4 GPS</div>
+                        <div class="card-description__desc">Redesigned from scratch and completely revised.</div>
+                    </div>
+                </div>
+                <div class="card__bottom">
```

Comment:
`card__bottom` both `div` children may also be `button` elements because user intends to click on them